packages/widget/src/state/wallets.ts

```diff
@@ -38,6 +38,7 @@ export type MinimalWallet = {
 };
 
 type WalletState = {
+  id?: string;
```

Comment:
we just need some kind of identifier, I think using one of the addresses associated to the wallet should be fine?